integration/shareddata/scalars.go

```diff
@@ -434,6 +434,31 @@ var ObjectIDKeys = &Values[primitive.ObjectID]{
 	},
 }
 
+// TopLevelFieldsIntegers contains documents with several top level fields with integer values.
+var TopLevelFieldsIntegers = &TopLevelValues[string]{
```

Comment:
Why did we add it to `scalars.go`? It looks like a composite. I guess, the name should also start with `Documents` like for other composites of similar nature.

Or, if you want to highlight  that it's a totally new thing (no `v` field), then maybe it should be in a new file (not scalars and not composites).

The test data itself looks super helpful! So far we've only tested `v` :D